app/Jobs/ProcessTransitionJob.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Services\TransitionProcessor;
+use App\Trader;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Str;
+use Imtigger\LaravelJobStatus\JobStatus;
+use Imtigger\LaravelJobStatus\Trackable;
+use Log;
+
+class ProcessTransitionJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels, Trackable;
```

Comment:
note trackable allows the jobstatus plugin to find this job, something L9 doesn't normally support